data/kahet/kahet missions.txt

```diff
@@ -137,27 +137,27 @@ mission "graveyard insights compatibility"
 
 event "graveyard insights"
 	planet Alix
-		add description `	 Here the Ka'het attacks were limited to the inhabited settlements; had they also destroyed the mining bases and the unmanned vessels, station Parii'het would have run out of supplies thousands of years ago.`
+		add description `	 Here the Ka'het attacks were limited to the inhabited settlements; had they also destroyed the mining bases and the unmanned vessels, Parii’het would have run out of supplies thousands of years ago.`
```

Comment:
This is true both here and in all other instances in the PR, but the correct apostrophe is this ', not this ’

Reply:
should be resolved now